### PR TITLE
Create a new entity when an agent comes online.

### PR DESCRIPTION
**Pull request by fguimond**
## What is this change?

Create a new Sensu entity when a new opamp agent connects to the WS server.

### CODE REVIEW

File: backend/opampd/protocol.go [hunk 1]
```diff
@@ -56,3 +67,31 @@ func (p *Protocol) OnAgentDisconnect(instanceUid string, disconnect *protobufs.A
 	//TODO implement me
 	return nil, fmt.Errorf("implement me")
 }
+
+func (p *Protocol) createEntity(instanceUid string, report *protobufs.StatusReport) error {
+	entity := &corev2.Entity{
+		EntityClass:        corev2.EntityAgentClass,
```
Comment (ccressent) [inline backend/opampd/protocol.go:73]: I don't think it's very important, but I think we said we would create a new entity class altogether, like `OT Agent` or something.

**Review by ccressent — APPROVED** [review]

File: backend/opampd/protocol.go (hunk 1 above)
Comment (portertech) [reply backend/opampd/protocol.go:73]: It does matter for filtering etc.

File: backend/opampd/protocol.go [hunk 2]
```diff
@@ -56,3 +67,31 @@ func (p *Protocol) OnAgentDisconnect(instanceUid string, disconnect *protobufs.A
...
+			Name:      instanceUid,
+			Namespace: entityNamespace,
+			Labels: map[string]string{
+				"opamp-instanceid": instanceUid,
+				"opamp-agent-type": report.AgentDescription.AgentType,
+			},
+			Annotations: nil,
+			CreatedBy:   "",
+		},
+		SensuAgentVersion: report.AgentDescription.AgentVersion,
```
Comment (portertech) [inline backend/opampd/protocol.go:92]: We shouldn't reuse this class and sensu agent specific fields. Something closure to a proxy entity makes more sense.

Comment (fguimond) [reply backend/opampd/protocol.go:92]: For now is it ok if I add this information as a `opamp-agent-version` label?

**Pull request merged**